composer/loggers/progress_bar_logger.py

```diff
@@ -103,21 +102,14 @@ def _start(self, state: State):
         if dist.get_global_rank() != 0:
             return
         assert self.is_train is not None, "self.is_train should be set by the callback"
-        if self.is_train:
-            total_steps = state.steps_per_epoch
-        else:
-            total_steps = 0
-            for evaluator in state.evaluators:
```

Comment:
Event.EVAL_START is now called for each evaluator, so the ProgressBarLogger does not need to sum over all evaluators anymore.